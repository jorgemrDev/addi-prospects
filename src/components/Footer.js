import React from "react";

export default function Footer() {
  return (
    <footer>
      <p>This site is created for technical test for ADDI</p>
      <p>&copy; JorgeMR</p>
    </footer>
  );
}
