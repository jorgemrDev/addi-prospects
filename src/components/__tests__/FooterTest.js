import React from "react";
import Footer from "../Footer";
import renderer from "react-test-renderer";

test("Render Footer correctly", () => {
  const component = renderer.create(<Footer></Footer>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
