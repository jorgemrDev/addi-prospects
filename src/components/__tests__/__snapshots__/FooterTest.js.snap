// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer correctly 1`] = `
<footer>
  <p>
    This site is created for technical test for ADDI
  </p>
  <p>
    Â© JorgeMR
  </p>
</footer>
`;
